introduction.md

```diff
@@ -1,2 +1,14 @@
 Hi I am Bill, I am practicing GitHub using code school.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?